README.md

```diff
@@ -1,2 +1,79 @@
-# lunar-payment-processor
-Flexible payment integration for Laravel, lunarphp, supporting multiple payment providers.
+# Lunar Stripe Payment Integration
+
+## Introduction
+This package provides a Stripe payment gateway integration for Lunar PHP. It allows you to handle payments seamlessly using Stripe's API, including creating payment intents and managing transactions.
+
+## Features
+- Stripe payment gateway integration
+- Support for card payments
+- Easy configuration and setup
+- Extensible architecture for additional payment methods
+
+## Minimum requirements
+- Lunar 1.x
+- A Stripe account with secret and public keys
+
+
+## Installation
```

Comment:
A stripeon mit kell létrehozni vagy beállítani arra nincs leírás. Pedig elég fontos lenne. És azt is el kellene valahol valahogyan magyarázzuk, hogy lényegében ez a package átirányítja a user-t a Stripe -ra és a fizetést ott kell eszközölje.